docs/source/locales/de/LC_MESSAGES/contribute.po

```diff
@@ -0,0 +1,49 @@
+# SOME DESCRIPTIVE TITLE.
```

Comment:
Where does "SOME DESCRIPTIVE TITLE" come from? Is it OK to leave this placeholder text?

Reply:
This was generated by sphinx. Trying to find out how to change this seems like a waste of time.